src/Trains.NET.Engine/Entity/Layout.cs

```diff
@@ -159,6 +159,7 @@ private void ResetArrays()
 
     public IEnumerator<IStaticEntity> GetEnumerator()
     {
+        if (_entities == null) yield break;
```

Comment:
Scary! What hit this?

Reply:
Great question, only hit it once, and didn't save the stack trace, added it as a safe-guard.